Ginger/GingerCore/Environments/Database.cs

```diff
@@ -255,61 +264,57 @@ public string GetConnectionString()
             bool res;
             res = false;
 
-            if (String.IsNullOrEmpty(ConnectionStringCalculated) == false)
+            if (String.IsNullOrEmpty(ConnectionStringCalculated))
             {
-                connStr = ConnectionStringCalculated.Replace("{USER}", UserCalculated);
-     
-                String deCryptValue = EncryptionHandler.DecryptString(PassCalculated, ref res, false);
-                if (res == true)
-                    { connStr = connStr.Replace("{PASS}", deCryptValue); }
-                else
-                    { connStr = connStr.Replace("{PASS}", PassCalculated); }
+                connStr = CreateConnectionString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'connStr'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5217045)